lib/widgets/components/discount_card.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class DiscountCard extends StatelessWidget {
+  final String title;
+  final String description;
+  final Widget iconStatus;
+
+  const DiscountCard({
+    super.key,
+    required this.title,
+    required this.description,
+    required this.iconStatus,
+  });
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      surfaceTintColor: Colors.greenAccent,
+      margin: const EdgeInsets.all(16),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(16),
+        side: const BorderSide(color: Colors.greenAccent, width: 1),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(16),
+        child: Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(right: 16),
```

Comment:
I think that it is much cleaner to put SizeBox with with in row instead of Padding right